docs/tutorial/snapcraft-guide.md

```diff
@@ -0,0 +1,181 @@
+# Snapcraft Guide (Ubuntu Software Center & More)
+
+This guide provides information on how to package your Electron application
+for the Ubuntu Software Center or any other Snapcraft environment.
```

Comment:
I'd suggest "how to package your Electron application for Linux."

While end users of Ubuntu, Fedora, and others install applications through GNOME Software, there are other storefronts that install snaps like the [MATE Software Boutique](http://www.omgubuntu.co.uk/2017/06/ubuntu-mate-new-software-boutique) and [KDE Discover](https://userbase.kde.org/Discover).

Reply:
Puh, tough one. I'm not really sure about that, given that my very own Snap currently doesn't run on Fedora _and_ that a guide on "How to package for Linux" should probably at least _mention_ `.deb` and `.rpm` files.

I'd rather leave it specific to Snapcraft, if that's okay with you – but I'm more than happy to let the other Electron maintainers overrule me.